test/unit/controllers/Admin.t.sol

```diff
@@ -149,15 +149,21 @@ contract MainnetControllerSetMaxSlippageTests is MainnetControllerAdminTestBase
             address(this),
             DEFAULT_ADMIN_ROLE
         ));
-        mainnetController.setMaxSlippage(makeAddr("pool"), 0.98e18);
+        mainnetController.setMaxSlippage(makeAddr("pool"), 0);
```

Comment:
Revert change

Reply:
Sure, but the value there is meaningless and unused given that the failure is not related.